client/src/components/AddressSearch.tsx

```diff
@@ -53,17 +54,18 @@ export function makeEmptySearchAddress(): SearchAddress {
   return {
     housenumber: "",
     streetname: "",
-    boro: "",
+    boro: null,
     bbl: "",
   };
 }
 
 function toSearchAddresses(results: GeoSearchResults): SearchAddress[] {
   return results.features.map((feature) => {
+    let boro = feature.properties.borough.toUpperCase() as Borough;
```

Comment:
@toolness is this kind of type assertion here kosher? Since the GeoSearch requester module we grab from the monorepo types the boro field as a string, I needed to "recast" this somehow to validate it as a proper borough name here. Saw guidance here: https://www.tutorialsteacher.com/typescript/type-assertion

Reply:
Ok sounds good! I have a feeling we might want to switch over to using some kind of type guard function when we start building out tests, but since that's out of scope for this PR I will leave as is for now.